### DISCUSSION TITLE
Suction Grippers

### DISCUSSION THREAD

**Discussion by clin-nexera** [Genesis-Embodied-AI/genesis-world#87] [Q&A]
Hello! Does the current version of genesis support suction grippers? 
If not, are there plans to support such a feature?

**Comment by zhouxian** [comment]
Not yet. Suction cup requires adding an additional constraint, which should be easy to add.

Yes we can support it!

**Reply by clin-nexera** [reply]
Will it simulate the physics of a suction cup such as wrench forces, seal quality, etc?

**Comment by alonks1234** [comment]
Having the ability to simulation suction grasps would definitely be really helpful! Is there a timeline in mind for this feature? Thank you!
